presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1263,6 +1263,10 @@ private void analyzeWindowFunctions(QuerySpecification node, List<Expression> ou
                     throw new SemanticException(NOT_SUPPORTED, node, "FILTER is not yet supported for window functions");
                 }
 
+                if (windowFunction.getOrderBy().isPresent()) {
+                    throw new SemanticException(NOT_SUPPORTED, windowFunction, "Window function with ORDER BY is not supported");
```

Comment:
should be part of `Add syntax support for aggregation_function(... ORDER BY ..) ` commit